tests/dummy/app/templates/public-pages/docs/the-list.hbs

```diff
@@ -149,6 +149,16 @@
   This is because it deserves an entire chapter of its own.
 </p>
 
+<h3>Prevent click propagation inside the list</h3>
+
+<p>
+ The options inside the list are selected with mousedown, a good way to prevent the selection is intercepting onmouseup and calling stopPropagation on the element "e.stopPropagation()"
```

Comment:
I'd add a note saying something: "This is particularly useful if you want to add some kind of link or button inside the list of option and prevent the option that contains it from being selected when the user clicks on them"